lib/tasks/database_updates/create_unique_site_banner_details_for_each_channel.rake

```diff
@@ -0,0 +1,43 @@
+namespace :database_updates do
+  desc "Create a unique site banner detail row for each channel that shares one on the publisher level"
+  task create_unique_site_banner_details_for_each_channel: :environment do
+    could_not_update = []
+    channels = Channel.where.missing(:site_banner)
+    channel_count = channels.count
+    puts "#{channels.count} channels have no site banner associated."
+
+    channels.each do |channel|
```

Comment:
Might want to do this in chunks to not run out of memory, since there could be a lot of channels. See [here](https://github.com/brave-intl/publishers/blob/staging/lib/tasks/database_updates/backfill_publisher_selected_wallet.rake#L43)